Tests/Bundle/DependencyInjection/EnqueueBridgeExtensionTest.php

```diff
@@ -39,6 +39,7 @@ public function testLoad()
     {
         $containerBuilderProphecy = $this->prophesize(ContainerBuilder::class);
         $containerBuilderProphecy->fileExists(Argument::any())->willReturn(false);
+        $containerBuilderProphecy->removeBindings('enqueue.messenger_transport.factory')->shouldBeCalledOnce();
```

Comment:
@sroze The tests fail because of this line.
When dependencies are installed with `prefer lowest` the `ContainerBuilder` doesn't have the `removeBindings()` method.
Is there a better way as to ensure `method_exist()` to fix the issue